content/toolsAndResources/nge/ngeBlocks.md

```diff
@@ -0,0 +1,1384 @@
+- **Contextual value** assigns the source of the data for any input set to a contextual value. The sources available depend on the data type selected for the input node. For example, choosing a Vector2 contextual value allows access to vertex IDs and choosing a Vector3 contextual value would offer vertex positions.
+- **Visible on frame** is a boolean which allows any input contained within a frame to be visible in the INPUTS list of the properties panel for a frame. Simply enable the boolean and then whenever a frame is selected - expanded or collapsed - any visible input can have its value changed directly in the frame properties panel.
+
+![Float input node](/img/tools/nge/FloatNode.jpg)
+
+### Float
+This input node represents a Float value rounded to four decimal places. The node contains the following properties which can be set in the properties panel. 
+
+#### Properties
+- **Min** provides the minimum value and if set, the node will create a slider between minimum and maximum values. For the slider to work correctly, ensure that the Min value is lower than Max value.
+- **Max** provides the maximum value and if set, the node will create a slider between minimum and maximum values. For the slider to work correctly, ensure that the Max value is higher than Min value.
+- **Value** is a user defined value that is either an input field or a combination of input field and slider if Min and Max are set. The value set in the field will be rounded to four decimal places by default.
+
+#### Outputs
+- **value** is the Float value set on the node. 
+
+![Integer node](/img/tools/nge/intNode.jpg)
+
+### Int
+This input node represents an integer value and contains the following properties which can be set in the properties panel. 
+
+#### Properties
+- **Min** provides the minimum value and if set, the node will create a slider between minimum and maximum values. For the slider to work correctly, ensure that the Min value is lower than Max value.
+- **Max** provides the maximum value and if set, the node will create a slider between minimum and maximum values. For the slider to work correctly, ensure that the Max value is higher than Min value.
+- **Value** is a user defined value that is either an input field or a combination of input field and slider if Min and Max are set. If a Float value is entered, any fractional value after the decimal will be ignored.
+
+#### Outputs
+- **value** is the integer value set on the node. 
+
+![Vector2 node](/img/tools/nge/vector2Node.jpg)
+
+### Vector2
+This input node represents a Vector2 value and contains the following properties which can be set in the properties panel. 
+
+#### Properties
+- **Value** displays the value set in the node. To change the value, click on the plus (+) icon to get the input fields for each channel. These values are rounded to three decimal places by default though the node only displays two decimal places.

```

Comment:
Same as above. I think we either need to set expectations that the values are rounded to the visible number of decimal places, or we have to allow the user to see the exact value when clicking in the input field. If we say the fields take more than 3 or 4 decimal places and then don't allow the user to ever see the whole number, this will feel like a bug.